tests/Pipfile

```diff
@@ -12,7 +12,7 @@ python_version = "3.6"
 
 [packages]
 
-shakedown = {git = "https://github.com/dcos/shakedown.git", ref = "1.4.11"}
+shakedown = {git = "https://github.com/dcos/shakedown.git", ref = "1.4.12"}
```

Comment:
The `Pipfile.lock` has to be updated as well.

Reply:
I did commit that too after we talked yesterday